BioForm/api/views.py

```diff
@@ -0,0 +1,143 @@
+import datetime
+import jwt
+from profiles.models import *
+from rest_framework import status, viewsets
+from rest_framework.response import Response
+from rest_framework.exceptions import AuthenticationFailed
+from rest_framework.views import APIView
+from .serializers import *
+from .permissions import IsJWTAuthenticated, IsUsersObject
+
+def serialize_user(user):
+    return {
+        'username': user.username,
+        'email': user.email,
+        'date_of_birth': user.date_of_birth,
+    }
+
+
+class RegisterView(APIView):
+    def post(self, request):
+        serializer = RegisterSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+        return Response(serializer.data)
+
+
+class LoginView(APIView):
+    def post(self, request):
+        email = request.data['email']
+        password = request.data['password']
+
+        user = Account.objects.filter(email=email).first()
+
+        if user is None:
+            raise AuthenticationFailed('User not found!')
+
+        if not user.check_password(password):
+            raise AuthenticationFailed('Incorrect Password!')
+
+        payload = {
+            'id': user.id,
+            'exp': datetime.datetime.utcnow() + datetime.timedelta(minutes=60),
+            'iat': datetime.datetime.utcnow()
+        }
+
+        token = jwt.encode(payload, 'secret', algorithm='HS256').decode('utf-8')
+
+        response = Response()
+        response.set_cookie(key='jwt', value=token, httponly=True)
+        response.data = {
+            'jwt': token,
+        }
+
+        return response
+
+
+class LogoutView(APIView):
+    def post(self, request):
+        response = Response()
+        response.delete_cookie('jwt')
+        response.data = {
+            'message': 'success',
+        }
+        return response
+
+
+class BioView(APIView):
+
+    permission_classes = [IsJWTAuthenticated]
+
+    def get(self, request):
+        try:
+            serializer = request.api_user
```

Comment:
I don't think you need to serialize the user in middleware and calling it `serializer` doesn't provide any useful information.